File: compose-app/shared/src/commonMain/kotlin/com/sales/app/domain/model/Module.kt
package com.sales.app.domain.model

data class Module(
    val id: Int,
    val name: String,
    val slug: String,
    val description: String?,
    val icon: String,
    val bgColor: String,
    val color: String,
    val isEnabled: Boolean,
    val sortOrder: Int
)

File: backend/app/Models/Module.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Module extends Model
{
    use HasFactory;

    protected $fillable = [
        'name',
        'slug',
        'description',
        'icon',
        'color',
        'bg_color',
        'is_enabled',
        'sort_order',
    ];

    //
}
